packages/docs/concepts/secrets.yaml

```diff
@@ -24,26 +24,22 @@ _ref:
         type: Markdown
         properties:
           content: |
-            The `secrets` object is a object that can be used to securely store sensitive information like passwords and API keys. Secrets can be accessed using the [`_secret`](/_secret) operator.
+            The `secrets` object is an object that can be used to securely store sensitive information such as passwords and API keys. Secrets can be accessed using the [`_secret`](/_secret) operator.
 
             The secrets object only exists on the backend server, and therefore the `_secret` operator can only be used in `connections` and `requests`.
 
-            We intend to support multiple secrets strategies in the future (for example AWS Secrets Manager or Docker secrets). Current secrets can only be set with environment variables.
-
             ## Environment variables strategy
```

Comment:
We should remove this heading. It looks like everything will still read fine without it.

Reply:
Change made in f0bddb86e11e71b9705c231eba66a8d71c8cdf56